tests/manage/z_cluster/test_drain_node_mon.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.resources.pod import get_mon_pods, get_pod_obj
+from ocs_ci.ocs.node import drain_nodes, schedule_nodes
+from ocs_ci.helpers.helpers import verify_pdb_mon
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier4a,
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+    ignore_leftovers,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier4a
+@skipif_external_mode
+@skipif_ocs_version("<4.6")
+@bugzilla("1959983")
+@ignore_leftovers
+class TestDrainNodeMon(ManageTest):
+    """
+    1.Get worker node name where monitoring pod run
+    2.Verify pdb status, disruptions_allowed=1, max_unavailable_mon=1
+    3.Drain node where monitoring pod run
+    4.Verify pdb status, disruptions_allowed=0, max_unavailable_mon=1
+    5.Verify the number of mon pods is 3 for (1400 seconds)
+    6.Respin  rook-ceph operator pod
+    7.Change node to be scheduled
+    8.Wait for mon and osd pods to be on running state
```

Comment:
Also check for all storage pods, Since it is node operation checking all other storage pods becomes necessary.